src/Tests/Framework/TestClient.cs

```diff
@@ -179,10 +179,19 @@ public static bool VersionSatisfiedBy(string range, ElasticsearchVersion version
 			string contentType = "application/json",
 			Exception exception = null)
 		{
-			var serializer = Default.Serializer;
-			var fixedResult = contentType == "application/json"
-				? serializer.SerializeToBytes(response)
-				: Encoding.UTF8.GetBytes(response.ToString());
+			byte[] fixedResult;
+
+			if (response.GetType() == typeof(string))
+			{
+				fixedResult = Encoding.UTF8.GetBytes((string)response);
+			}
+			else
+			{
+				var serializer = Default.Serializer;
+				fixedResult = contentType == "application/json"
+					? serializer.SerializeToBytes(response)
+					: Encoding.UTF8.GetBytes(response.ToString());
+			}
```

Comment:
Rather then supporting strings can we use anonymous c# objects in the tests for the expected json?

Reply:
I went down that road originally, but I had to end up using JObject, which might have looked messier.

Reply:
Why did you need to use jobject?

Reply:
Property name starts with a '.'